internal/pkg/node/datastructure.go

```diff
@@ -1,5 +1,11 @@
 package node
 
+import (
+	"strings"
+
+	"github.com/ghodss/yaml"
```

Comment:
Is there are reason for using this yaml implementation and not "gopkg.in/yaml.v2" or was it just the insert from the editor?